apis/core/validation/installation_test.go

```diff
@@ -233,10 +233,28 @@ var _ = Describe("Installation", func() {
 						},
 					},
 				},
-				Targets: []core.TargetImportExport{{
-					Name:   "foo",
-					Target: "fooTarget",
-				}},
+				Targets: []core.TargetImport{
+					{
+						Name:   "foobaz",
+						Target: "fooTarget",
+					},
+					{
+						Name:                "barbaz",
+						TargetListReference: "foobaz",
+					},
+					{
+						Name: "baz",
+						Targets: []string{
+							"t1",
+							"t2",
+							"t3",
+						},
+					},
+					{
+						Name:    "fb",
+						Targets: []string{},
+					},
+				},
```

Comment:
the test only covers duplicated target names but not a data import and target with the same name?
Or do I miss something here?

Reply:
The lines you have linked belong to the test that "should pass if imports are valid". It does actually not contain any duplicates (I needed to change some import names, since it did include cross-type duplicates before and wouldn't pass anymore after my changes to the validation).
The test that checks for duplicates starts in line 246.